flatdata-rs/lib/src/arrayview.rs

```diff
@@ -66,7 +66,7 @@ mod test {
         assert_eq!(vec.len(), 3);
         assert_eq!(vec[0].x(), 10..20);
         assert_eq!(vec[1].x(), 20..30);
-        assert_eq!(vec[2].x(), 30..0);
+        assert!(vec[2].x().is_empty());
```

Comment:
This is not the same check, is it? A range can also be "non-nonsensical", that is going backwards